scripts/prepare-release

```diff
@@ -50,9 +50,13 @@ version_json="$(jq --arg v "${new_version#v}" '."vast-version-fallback" = $v' ve
 echo -E "${version_json}" > version.json
 
 # If the new version is not a release candidate, change mentions of the version
-# in README.md as well.
+# in README.md as well, and create a new docs version.
 if [[ "${new_version}" != *"-rc"* ]]; then
   perl -i -pe "s/${last_version}/${new_version}/g" "${source_dir}/README.md"
+  pushd "${source_dir}/web"
+  yarn install --frozen-lockfile
+  yarn docusaurus docs:version "${new_version}"
```

Comment:
After calling this, I get new files in the repo:

```
	web/versioned_docs/
	web/versioned_sidebars/
	web/versions.json
```

Where are they added to the repo? Does that need to happen as well?

Reply:
The prepare-release script adds them to the repo, commits them, and creates a PR. That's already part of the script.